web/modules/custom/custom_routes/src/Routing/CustomRoutesSubscriber.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Drupal\custom_routes\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Alters the requirements of the route.
+ */
+class CustomRoutesSubscriber extends RouteSubscriberBase {
+
+  /**
+   * Alters roles, permission and adds custom access checks.
+   *
```

Comment:
Missing parameter type in comment.

Reply:
Parameter type already added in line 16 \Symfony\Component\Routing\RouteCollection